lib/xcpretty/syntax.rb

```diff
@@ -12,20 +14,22 @@ module XCPretty
   module Syntax
     def self.highlight(snippet)
       return snippet.contents unless Rouge
+
       highlight_with_formatter(snippet, Rouge::Formatters::Terminal256.new)
     end
 
     def self.highlight_html(snippet)
       return snippet.contents unless Rouge
+
       highlight_with_formatter(snippet, Rouge::Formatters::HTML.new)
     end
 
     def self.highlight_with_formatter(snippet, formatter)
-      if snippet.file_path.include?(':')
-        filename = snippet.file_path.rpartition(':').first
-      else
-        filename = snippet.file_path
-      end
+      filename = if snippet.file_path.include?(':')
```

Comment:
Maybe use `?:` operator to use less vertical space.

Reply:
rubocop doesn't like multiline ternary